website/docs/guides/mesh-qs.md

```diff
@@ -0,0 +1,563 @@
+
+```sql
+select * from {{ ref('fct_orders', v=1) }}
+select * from {{ ref('fct_orders', v=2) }}
+select * from {{ ref('fct_orders') }}
+```
+
+## Add dbt Cloud Job in the downstream project
+Before proceeding, commit & merge your changes in both the “Jaffle | Data Analytics” and “Jaffle | Finance” projects. 
+
+A member of the Finance team would like to schedule a dbt Cloud Job their customer payment journey analysis immediately after the Data Analytics team refreshes their pipelines.
+
+In the “Jaffle | Finance” project, go to the Jobs page (“Deploy” -> “Jobs”) and click “Create job” -> “Deploy job”. Add a name for the job, then scroll to the bottom and configure the job to “Run when another job finishes”. Select the upstream job, like so:
+<!-- <Lightbox src="/img/guides/dbt-mesh/trigger_on_completion.png" title="Trigger job on completion" /> -->
+
+Click “Save”. Let’s verify this works as we expect. Go to the “Jaffle | Data Analytics” Jobs page, select the “Daily Job”, and click “Run Now”. Once this job completes successfully, go back the “Jaffle | Finance” Jobs page. Boom! The Finance team’s job was triggered automatically.
+
+This makes it dead simple to stay in sync with the upstream tables and removes the need for more sophisticated orchestration skills (e.g. coordinating jobs across projects via an external orchestrator).
+
+## View deprecation warning
+
+How would a finance team member know how long they have to migrate over from fct_orders_v1 to fct_orders_v2?
+
+Let’s check. Head back to the “Develop” page in the “Jaffle | Finance” project. Edit the cross project ref to use v=1 in “models/marts/agg_customer_payment_journey.sql”:
+
+<File name='models/core/agg_customer_payment_journey.sql'>
+
+```sql
+
+with stg_payments as (
+    select * from {{ ref('stg_payments') }}
+),
+
+fct_orders as (
+    select * from {{ ref('analytics', 'fct_orders', v=1) }}
+),
+
+final as (
+    select 
+        days_as_customer_at_purchase,
+        -- we use the pivot macro in the dbt_utils package to create columns that total payments for each method
+        {{ dbt_utils.pivot(
+            'payment_method',
+            dbt_utils.get_column_values(ref('stg_payments'), 'payment_method'),
+            agg='sum',
+            then_value='amount',
+            prefix='total_',
+            suffix='_amount'
+        ) }}, 
+        sum(amount) as total_amount
+    from fct_orders
+    left join stg_payments using (order_id)
+    group by 1
+)
+
+select * from final
+
+```
+
+</File>
+
+Then, commit & merge the change. Go to the “Deploy -> Jobs” page and click “Run Now” on the Finance Job. We’ll see that the the agg_customer_payment_journey model builds but surfaces a warning with the deprecation date.
+
+<!-- <Lightbox src="/img/guides/dbt-mesh/deprecation_date_warning.png" title="Deprecation date warning" /> -->
+
+## View lineage with dbt Explorer
+
```

Comment:
if you'd like, it might be mice to wrap the last bits in a confetti component for a nice little easter egg! we have one in the core to cloud guide ([see here and click anywhere on the page](https://docs.getdbt.com/guides/core-to-cloud-1?step=10)).

to do this, just wrap the content inside the `<ConfettiTrigger></ConfettiTrigger>`  component!

Reply:
love it!